app/modes/initmodes.F90

```diff
@@ -123,6 +131,9 @@ module modes_initmodes
     integer :: divideandconquer = 2
     integer :: relativelyrobust = 3
 
+    ! GPU accelerated solver using MAGMA
+    integer :: magma_evd = 4
```

Comment:
```suggestion
    integer :: magmaEvd = 4
```
Likewise for `call gpu_evd()` etc., but I'm fine with with keeping the current version if this follows MAGMA conventions.

Reply:
I suggest a follow on PR to change gpu_evd to magmaHeevd and likewise gpu_gvd -> magmaHegvd

Reply:
Works for me :+1:.